buildSrc/src/main/kotlin/CopyReadme.kt

```diff
@@ -0,0 +1,58 @@
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.StandardCopyOption
+import java.nio.file.StandardOpenOption
+
+const val READ_ME_FILE = "README.md"
+
+open class CopyReadme : DefaultTask() {
+
+  init {
+    group = "documentation"
+    description = "Copies the readme to a place where it can be converted with other markdown files"
+  }
+
+  @TaskAction
+  fun run() {
+    val markdownPath = createMarkdownDirectory(project)
+    val readmeMarkdown = Paths.get(
+      markdownPath.toString(),
+      READ_ME_FILE
+    )
+    Files.copy(
+      Paths.get(project.rootDir.absolutePath, READ_ME_FILE),
+      readmeMarkdown,
+      StandardCopyOption.REPLACE_EXISTING
+    )
+
+    val trimmedREADME = Files.readAllLines(readmeMarkdown)
```

Comment:
I figured it be easier to omit the stuff from the markdown before converting it to HTML, that way I had less stuff to do after the conversion.